libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -354,6 +354,8 @@ static void eventCallback( MQTTContext_t * pContext,
     }
     else
     {
+        TEST_ASSERT_NOT_EQUAL( MQTT_PACKET_TYPE_PINGRESP, pPacketInfo->type );
```

Comment:
Don't really think this should be here, since it's possible we want to test MQTT_ReceiveLoop(), which will pass PINGRESPs back to the callback. I'd remove this from this file